code/src/components/Item.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import todos from 'reducers/todos';
+import CustomCheckBox from 'components/CustomCheckbox';
+import { ItemList, RemoveItemButton } from 'styling/ItemStyling';
+
+const Item = ({ id, item, isComplete }) => {
+
+    const dispatch = useDispatch();
+
+    const onCheckBoxClick = () => {  
+        dispatch(todos.actions.changeItem({id}))
+    }
+
+    const onItemDelete = () => {
+        dispatch(todos.actions.removeItem(id));
+    }
+
+    return (
+        <ItemList>
+             <CustomCheckBox 
+                isChecked={isComplete} 
+                onChangeHandler={onCheckBoxClick}/>
+             {item}
+             <RemoveItemButton onClick={onItemDelete}>
+                   <span>🗑</span> 
```

Comment:
Have a look into the styling of the delete bin as they are a bit off.

![Screenshot 2020-12-02 at 11 10 50](https://user-images.githubusercontent.com/62376840/100859044-0abc1c80-348f-11eb-8141-d4b79796207d.png)